akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/common/model/ScrapyRT.scala

```diff
@@ -0,0 +1,22 @@
+package com.tapiporla.microservices.retrievers.common.model
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import spray.json.DefaultJsonProtocol
+
+
+case class ScrapyRTRequest(
+                            spider_name: String,
+                            start_requests: Boolean,
+                            parameters: Map[String,String]
+                          )
+
+case class ScrapyRTResponse(
+                           status: String,
+                           spider_name: String,
+                           items: Seq[Map[String,String]]
+                           )
+
+trait ScrapyRTDefaultProtocol extends SprayJsonSupport with DefaultJsonProtocol{
```

Comment:
I'm more fan of importing ```DefaultJsonProtocol``` inside the trait than extending it (just in case you wanted to mix-in some ```ScrapyRTProtocol``` with ```SomeOtherStuffProtocol``` so there is no clash of common ancestors)

Reply:
I think that ScrapyRT will be always in JSON. Scrapyrt don't support more formats, and due the last activity in the project, I think JSON is the default "REST" Api format

Reply:
That's not what I meant. I just suggested to do something like
```scala
trait ScrapyRTDefaultProtocol { _: SprayJsonSupport with DefaultJsonProtocol =>
  //your formatters
}
```
just in case you had another trait with more JsonFormats:
```scala
trait SomeOtherStuffProtocol{ _: SprayJsonSupport with DefaultJsonProtocol =>
}
```
and you wanted to uniquely instantiate a JsonProtocol when deploying the REST API:
```scala
object JsonProtocol extends SprayJsonSupport with DefaultJsonProtocol
  with ScrapyRTDefaultProtocol
  with SomeOtherStuffProtocol
```

But I didn't check if this applies to current case